packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -124,6 +124,54 @@ module.exports = ({ types: t }) => {
         ],
       },
 
+      LogicalExpression(path) {
```

Comment:
It feels these simplifications should be done in a post-order traversal (i.e. on `exit`) -- does this resolve properly for nested conditions?

e.g. `(3 || foo()) || bar()` should simplify to just `3`.